appveyor.yml

```diff
@@ -12,6 +12,12 @@
 #*                                                                        *
 #**************************************************************************
 
+branches:
+  only:
+    - /4\.*/
+    - /5\.*/
+    - trunk
```

Comment:
Does this _definitely_ work - I have a feeling that when [PR#11331](https://github.com/ocaml/ocaml/pull/11331) was created, there was an issue with trying to do the same for AppVeyor?